### ISSUE TITLE
Problems with installation

### ISSUE THREAD

**Issue by TimoRuetten** [gre/gl-react#124]
### *bug report*

I tried to get the examples work but there seems to be a problem with my installation. I redid the installation now several times but none of them seems to work.

After the installation for iOs I tried - just for fun - to build the app via XCode and this error failed the build:

```
Failed to load:
#import <React/RCTJSCExecutor.h>
In: EXGLView.m
```

What I have done:
react-native init
npm i gl-react-native --save
npm i gl-react --save
moved the .xcode file from node_modules/gl-react-native to Libraries of my app in xcode
added in **app** => _targets_ **app** libRNGL.a
inserted your helloGL example.
Result: Just a red dot
inserted the animated example
Result: Just a white screen

Am I missing smth ?

#### library version

```bash
├── gl-react@3.6.0 
└── gl-react-native@3.6.0 
```

A screenshot of my settings:
<img width="1022" alt="screen shot 2017-06-19 at 10 02 25" src="https://user-images.githubusercontent.com/7010059/27266817-850c8688-54d6-11e7-9cc8-47e96407f32e.png">

This error got throwed in remote console:

> Warning: Native component for "EXGLView" does not exist

**Comment by DasPez** [comment]
I tried to run the examples on Android (7.1.2) using android-ndk-r10e and Linux. Maybe the NDK is too old... NDK r15 does not include the `arm-linux-androideabi-4.8` toolchain that is needed for gl-react-native.
![screenshot_20170623-120353](https://user-images.githubusercontent.com/23032803/27477621-fb7fe6f0-580c-11e7-8d79-6ef1ece2bc40.png)

**Comment by Josephmaxim** [comment]
See [**Issue #117: 'EXGLView' has no propType for native prop 'EXGLView.accessibilityLabel' of native type 'String'**](https://github.com/gre/gl-react/issues/117)

**Comment by gre** [comment]
sorry guys, was pretty busy the past weeks, i'm back! going to tackle that in the week

**Comment by aaronbuchanan** [comment]
eager for a fix here, looks like the `...ViewPropTypes` bit @gre referred to might be the fix here?
 It seemed to help when I messed around locally anyhow.

**Comment by kelset** [comment]
To get back to @TimoRuetten original issue, I suggest you have a look at [this other issue](https://github.com/gre/gl-react/issues/106#issuecomment-299928891) because I think the workaround is the same.

**Comment by gre** [comment]
damn, i don't know what's happening in the project build, but half of the time, the `yarn` command randomly hangs forever (I need to investigate on that. not sure what's happening, if it's a yarn registry issue or something else).

anyway, I'm trying to update libs step by step, I was able to upgrade cookbook and also to upgrade Expo in cookbook-expo so at least gl-react-expo should work with latest libs.
but now I'm stuck trying to upgrade yet `cookbook-rn` pkg to latest React Native.. for some reason React gets no longer resolved after that upgrade: 

I hope to pass this config step soon so i can fix the actual bug..

**Comment by kelset** [comment]
I see, let us know how it goes or if you need any help 💪

Btw, related to the `yarn` issues; Expo "suggests/forces" you to install and use `npm@4.6.1`, have you tried using that? (I'm going to try soon)

**Comment by gre** [comment]
looks like a fix will probably be around replacing `#import <React/RCTJSCExecutor.h>` by `#import <cxxreact/JSCExecutor.h>` – wondering if/how expo fixed that

**Comment by kelset** [comment]
Looks like in expo they [didn't fix it](https://github.com/expo/expo/blob/79069d41b6278e773f5f0c03dde6d7cb22bea691/ios/Exponent/Versioned/Modules/Api/GL/EXGLView.m#L7), but the reason is probably related to the fact that lately expo has been running its own version of [react-native](https://github.com/expo/react-native).

**Comment by gre** [comment]
ok so for my yarn problem, apparently you should stay on Node 6, some issues seem to appear with lerna/yarn and Node 8 .. this might be related to https://github.com/stackgl/headless-gl/issues/111 actually

**Comment by gre** [comment]
good news, after bunch of native changes, i'm able to run gl-react cookbook again, going to test some things a bit to make sure it works

**Comment by gre** [comment]
fixed @3.7.0
